init.lua

```diff
@@ -96,7 +102,14 @@ end
 
 
 
+local init_delay = 1
```

Comment:
wouldn't it make more sense to ensure that ``rocket_delay`` is within sane limits? Let's say 1...65000.
Then ``init_delay`` is not needed and ``rocket_delay`` can be used throughout code.